.github/workflows/test.yml

```diff
@@ -30,11 +30,18 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Install GCC 10.2 Linux
```

Comment:
Does this impose dependency on gcc v10.2 ? Why not default gcc from github actions image ?

Reply:
https://en.cppreference.com/w/cpp/algorithm/execution_policy_tag -> inline constexpr std::execution::unsequenced_policy unseq is available since C++20, even though gcc 9.0 supports this C++ 20 feature it compiles with an error of missing include (somewhere deep in standard library)